geojson_pydantic/features.py

```diff
@@ -27,6 +27,19 @@ class Config:
 
         use_enum_values = True
 
+    @validator("bbox", pre=True)
+    def check_bbox(cls, bbox: BBox) -> BBox:
+        if len(bbox) == 6:
+            if bbox[0] > bbox[3] or bbox[1] > bbox[4] or bbox[2] > bbox[5]:
+                raise ValueError("BBox must be in the form [west, south, bottom, east, north, top]")
+        elif len(bbox) == 4:
+            if bbox[0] > bbox[2] or bbox[1] > bbox[3]:
+                raise ValueError("BBox must be in the form [west, south, east, north]")
+        else:
+            raise ValueError("BBox must be in the form [west, south, east, north] or [west, south, bottom, east, north, top]")
+        return bbox 
+
+
```

Comment:
would you be able to add some tests for this 🙏

Reply:
That was fast! Yes, absolutely

Reply:
The tests are pretty simple, but I think they get the job done. Let me know if I can do anything else!